docs/introduction-to-smart-contracts.rst

```diff
@@ -387,7 +389,8 @@ the topmost 16 elements to the top of the stack or swap the
 topmost element with one of the 16 elements below it.
 All other operations take the topmost two (or one, or more, depending on
```

Comment:
``the topmost two (or one, or more`` :-)? Maybe ``All other operations take the elements they require from the top of the stack and push the result onto the stack``.

Reply:
Feel free to create a PR :)